### PR TITLE
v0.33.0 - addition of sticky attr to header, addition of header button styles

### PR DESCRIPTION
**Pull request by sonnyprince**
__*Addition of header button styles and data attribute to header*__

- Addition of header button styles:
screen shot 2018-08-22 at 13 00 50

- Addition of data attribute:
screen shot 2018-08-22 at 13 01 06

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
## Browsers Tested

- [x] Chrome
- [x] Edge
- [x] Internet Explorer 11
- [x] Mobile

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: src/scss/partials/_button.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* Header button Styling
+ * =================================
+ * Styles relating to button (like mobile filter button) within header
+ */
+
+ .c-headerButton {
+    border: 0;
+    outline: 0;
+    padding: 0;
+    font-size: 0;
```
Comment (DamianMullins) [inline src/scss/partials/_button.scss]: Should this element be added to the header template as well? 

Also, are some of these styles here to hide text from the user?

File: src/templates/header/index.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {{> skip-to }}
 
-<header class="c-header">
+<header class="c-header" data-sticky-element>
```
[the remark this answers is not in the document]
Comment (sonnyprince) [reply src/templates/header/index.hbs]: no, The sticky module has to be called and have the elements passed through

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.33.0
+------------------------------
+*August 22, 2018*
+
+### Added
+- `data-sticky-element` attribute to header
+- Addition of `c-headerButton` styles
```
Comment (ashleynolan) [inline CHANGELOG.md]: Need to update this line now

**Comment by sonnyprince** [comment]
@DamianMullins @ashleynolan - updated - ticks are much appreciated! 🙏

File: src/scss/index.scss [hunk 4]
```diff
@@ -74,3 +88,25 @@ $header-border-color              : $red !default;
         }
     }
 
+    // Header button Styling
+
+    .c-header-button {
+        border: 0;
+        outline: 0;
```
Comment (DamianMullins) [inline src/scss/index.scss]: Do you definitely want to remove the outline around this element?

File: src/scss/index.scss [hunk 5]
```diff
@@ -63,6 +64,19 @@ $header-border-color              : $red !default;
             z-index: zIndex(high);
         }
     }
+
+    // Styles for a sticky header on mobile
+    @include media('<mid') {
+        &.is-sticky {
+            left: 0;
+            top: -60px;
+            position: fixed;
```
Comment (ashleynolan) [inline src/scss/index.scss]: Late to the game – just to check – doesn't `is-sticky` helper already add this line?

**Pull request merged**